sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj

```diff
@@ -27,6 +27,7 @@
     <ProjectReference Include="..\..\Azure.Storage.Blobs\src\Azure.Storage.Blobs.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)AppContextSwitchHelper.cs" LinkBase="SharedCore" />
```

Comment:
`SharedCore` or `Shared\Core` ? some links below are not consistent.

Reply:
All the other links besides `AuthorizationChallengeParser` are `SharedCore`